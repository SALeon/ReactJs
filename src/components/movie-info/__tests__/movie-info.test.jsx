import React from 'react';
import { shallow } from 'enzyme';
import MovieInfo from '../index';

const movieInfo = {
  poster_path: 'poster',
  release_date: '40-80',
  title: 'title',
  overview: 'overview',
};

it('render the MovieInfo component', () => {
  const renderedMovieInfo = shallow(<MovieInfo movieInfo={movieInfo} />);
  expect(renderedMovieInfo).toMatchSnapshot();
});
