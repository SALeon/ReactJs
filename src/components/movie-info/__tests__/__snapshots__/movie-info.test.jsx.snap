// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the MovieInfo component 1`] = `
<div
  className="movie-info"
>
  <img
    alt="movie img"
    className="movie-info__image"
    src="poster"
  />
  <div
    className="movie-info__description"
  >
    <h2
      className="movie-info__title"
    >
      title
    </h2>
    <span
      className="movie-info__data"
    >
      40
    </span>
    <p
      className="movie-info__description-text"
    >
      overview
    </p>
  </div>
</div>
`;
